RasimKhusaenov/my_own_blog
# Install bootstrap correctly

### Summary

- Installed bootstrap as node module
- Changed primary and secondary colors

### How it works
Before:
|:--:|
| ![image](https://user-images.githubusercontent.com/35406008/179747302-8c953dbd-d026-40ea-a945-7ec337e694a9.png) |

After:
|:--:|
| ![image](https://user-images.githubusercontent.com/35406008/179746947-7b38cdc6-4a31-48e8-80cd-e58211921ca8.png) |

### Test plan

List of steps to manually test introduced functionality:

- Go to https://my-own-blog-pr-75.herokuapp.com/
- Make sure that primary objects greeny

### Deploy notes

Run via console:
```
yarn install --check-files
```

### Checklist:


### References
- https://getbootstrap.com/docs/5.0/getting-started/webpack/
